packages/vite/src/utils/generator-utils.ts

```diff
@@ -315,9 +356,16 @@ export function writeViteConfig(tree: Tree, options: Schema) {
 
   let viteConfigContent = '';
 
+  const testConfig = `test: {
+    globals: ${options.inSourceTests ? false : true},
+    environment: 'jsdom',
```

Comment:
Because we use `jsdom` here, we also need to install `jsdom` in the workspace. Otherwise we run into a missing dep error. I'd the package to the `init` generator since the intention is to use vitest for all vite projects.